homeassistant/components/hydrawise/entity.py

```diff
@@ -24,24 +24,38 @@ def __init__(
         coordinator: HydrawiseDataUpdateCoordinator,
         description: EntityDescription,
         controller: Controller,
+        *,
         zone: Zone | None = None,
```

Comment:
And I mean literally just pass the id here instead

Reply:
Got it. Changed.